### DISCUSSION TITLE
Blogpost 8

### DISCUSSION THREAD

**Discussion by LeonRother** [LeonRother/SE_Platzverwaltung#9] [General]
In dieser Woche haben wir das Software Architecture Document (SAD) erstellt. Dieses dokumentiert die Architektur unseres Projekts und dient als Leitfaden für die weitere Entwicklung. Das SAD enthält detaillierte Informationen über die Systemkomponenten, deren Schnittstellen und die zugrunde liegende Infrastruktur.

Das Dokument ist unter folgendem Link verfügbar:
[SAD](https://github.com/LeonRother/SE_Platzverwaltung/blob/8302ca0721b8c0ad09c5c3b1a8b8f9476ae8684d/docs/SAD.md)

Viele Grüße
Leon & Team Deskflex

**Comment by Yahya-EzEd** [comment]
Moin

Euer Software Architecture Document sieht wirklich gut aus und scheint sehr durchdacht zu sein! Wie seid ihr bei der Dokumentation eurer Architektur vorgegangen, um sicherzustellen, dass alle wichtigen Details abgedeckt sind?

LG
Restaurant Reservierung

**Comment by Mimikry2** [comment]
Moin,

Die Vorlage für das Software Architecture Document bietet eine gut strukturierte und umfassende Grundlage zur Dokumentation von Softwarearchitekturen. Besonders hervorzuheben ist die klare Gliederung in verschiedene Sichten wie Process View und Logical View, die eine detaillierte Beschreibung der Systemarchitektur ermöglichen. Die Praxisnähe zeigt sich in der Beschreibung von Frontend- und Backend-Komponenten, die moderne Softwareentwicklungsansätze widerspiegeln.

Allerdings könnten die Einleitung und Zielgruppe genauer definiert werden, um den praktischen Nutzen des Dokuments für unterschiedliche Rollen im Projekt zu verdeutlichen.

Viele Grüße,
Tim, Dennis, Nico
